main.go

```diff
@@ -101,12 +93,8 @@ func main() {
 		select {
 		case <-newLeader:
 			tryReload()
-		case err := <-dnsErr:
-			handleServerErr("DNS server", err)
-		case err := <-httpErr:
-			handleServerErr("HTTP server", err)
-		case err := <-zkErr:
-			handleServerErr("ZK watcher", err)
+		case err := <-errch:
+			logging.Error.Fatal(err)
```

Comment:
It doesn't matter because `logging.Error.Fatal` will terminate the whole process.